cpp/include/cuvs/neighbors/common.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, NVIDIA CORPORATION.
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
I'm not sure if the copyright check is working correctly in cuvs. Can we file an issue to address it later?